commands/geosearch.md

```diff
@@ -47,5 +47,5 @@ When additional information is returned as an array of arrays for each item, the
 GEOADD Sicily 13.361389 38.115556 "Palermo" 15.087269 37.502669 "Catania"
 GEOADD Sicily 12.758489 38.788135 "edge1"   17.241510 38.788135 "edge2" 
 GEOSEARCH Sicily FROMLONLAT 15 37 BYRADIUS 200 km ASC
-GEOSEARCH Sicily FROMLONLAT 15 37 BYBOX 400 400 km ASC
+GEOSEARCH Sicily FROMLONLAT 15 37 BYBOX 400 400 km ASC WITHCOORD WITHDIST WITHHASH
```

Comment:
I'd remove the `WITHHASH` this isn't useful enough to appear as an example (might be confusing).
```suggestion
GEOSEARCH Sicily FROMLONLAT 15 37 BYBOX 400 400 km ASC WITHCOORD WITHDIST
```